trulens/nn/models/pytorch.py

```diff
@@ -64,8 +68,14 @@ def __init__(
         self._layernames = list(layers.keys())
         self._tensors = list(layers.values())
 
+        if 0 == len(self._tensors):
```

Comment:
nit: swap the operands to be consistent with the rest of code:
len(self._tensors) == 0

Reply:
I'd suggest to swap all others to "0 == ".

Reply:
Even our inequalities are written in a way variable > constant. Not sure if there's a good reason to swap them all

Reply:
Generally "constant == variable" is preferable given the "constant = variable" typo becomes an error instead of a silent bug with "variable = constant" but it looks like python catches that for you so will switch.

Reply:
Done.